ntc_templates/templates/alcatel_aos_show_system.textfsm

```diff
@@ -0,0 +1,27 @@
+Value DESCRIPTION (.+,.+,[^,]+)
+Value OBJECT_ID ([^,]+)
+Value UP_TIME (\d+ days \d+ hours \d+ minutes and \d+ seconds)
+Value CONTACT (.+,(?: https://\S+)?)
+Value NAME ([^,]+)
+Value LOCATION ([^,]+)
+Value Service (\d+)
+Value DATE_TIME (\S+\s*\S+\s*\d+\s*\d+\s*\d+:\d+:\d+\s*(\(\w+\)))
+Value AVAILABLE (\d+)
+Value COMMENTS ([^,]+)
+
+Start
+  ^\s*System:$$
+  ^\s*Description:\s*${DESCRIPTION},$$
+  ^\s*Object\s*ID:\s*${OBJECT_ID},$$
+  ^\s*Up\s*Time:\s*${UP_TIME},$$
```

Comment:
```suggestion
  ^\s*Up\s*Time:\s*${UPTIME},$$
```

Match capture group change (as noted above in the file) to the common occurrence.